### DISCUSSION TITLE
How to set/schedule a date for a post to be created

### DISCUSSION THREAD

**Discussion by GodsonAddy** [sequelize/sequelize#15627] [Help & Questions] [answered]
So I'm creating a blog app and want the user to set a date and time for his/her blog post to be created. So instead of the post being created at let's say DATE.NOW(), the post is created at a designated date and time

So example: 

`
BlogPost.create({
title,
content,
createdAt: "01/02/2023" // The user gets to set their own date for the post. So the date will be in the future.
})
`

I hope my question is understandable. Thanks

**Comment by ephys** [comment, marked as answer]
`createdAt` is intended to mean the timestamp at which the row was added to the table. I recommend using a separate field for publication date, such as `publishedAt`

If you really want to use `createdAt` for this, here is how you can disable sequelize's `createdAt`: https://sequelize.org/docs/v6/core-concepts/model-basics/#timestamps, which you can then replace with your own
